### DISCUSSION TITLE
more domains (light) for entity node

### DISCUSSION THREAD

**Discussion by Syntoxr** [zachowj/node-red-contrib-home-assistant-websocket#521] [Feature Request]
This weekend I wanted to add some Christmas lights via NR and wondered why the entity node only has the options for `sensor`, `binary_sensor `or `switch`. I ended up using a template light, but this doesn't feel nice as I have twice the number of entities. I'm wondering what's the reason for only having those three available?

**Comment by Syntoxr** [comment]
Nothing? I'm just interested if it would make sense to add other domains, or why it wouldn't.
Thanks

**Reply by zachowj** [reply]
Because no one has gotten around to adding them.

**Reply by Syntoxr** [reply]
Ah, thanks for the response and info

**Comment by skylord123** [comment]
Yeah, I just ran into this. Would really like support for creating lights in Node-RED to reflect in HA. 

Wanted to use it for controlling the backlight brightness & color for my laptop. Will have to go another route for now.
